packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -83,7 +84,7 @@ export const IcaAccountHolderI = M.interface('IcaAccountHolder', {
   undelegate: M.call(M.arrayOf(DelegationShape)).returns(VowShape),
 });
 
-/** @type {{ [name: string]: [description: string, valueShape: Pattern] }} */
+/** @type {{ [name: string]: [description: string, valueShape: Matcher] }} */
```

Comment:
Why change the type from `Pattern` to `Matcher`? Do you intend to exclude other patterns that are not themselves matchers, even if they contain matchers?

Likewise for other similar changes

Reply:
> Why change the type from Pattern to Matcher? 

because `makeRecordKit` accepts only a Matcher for the valueShape: https://github.com/Agoric/agoric-sdk/blob/4d54c56d1bba635e926d43fcc2f0c6788a88b6e2/packages/zoe/src/contractSupport/recorder.js#L148-L149

> Do you intend to exclude other patterns that are not themselves matchers, even if they contain matchers?

For now. https://github.com/Agoric/agoric-sdk/blob/4d54c56d1bba635e926d43fcc2f0c6788a88b6e2/packages/zoe/src/contractSupport/recorder.js#L261-L271

Reply:
`Pattern` is no longer `any` (thanks to you IIRC)

> `@typedef {Exclude<Passable, Error | Promise>} Pattern`

Reply:
Yes. From https://github.com/endojs/endo/pull/2238

Reply:
Ah yes, we can fix that now: 6b4182963cbc87b6eb22b77833e420fa35bb2da6